src/Bridges/Constraint/Constraint.jl

```diff
@@ -64,12 +64,12 @@ const RSOCtoPSD{T, OT<:MOI.ModelLike} = SingleBridgeOptimizer{RSOCtoPSDBridge{T}
 include("indicator.jl")
 
 """
-    add_all_bridges(bridged_model, T::Type)
+    add_all_bridges(bridged_model, ::Type{T})
 
 Add all bridges defined in the `Bridges.Constraint` submodule to
 `bridged_model`. The coefficient type used is `T`.
 """
-function add_all_bridges(bridged_model, T::Type)
+function add_all_bridges(bridged_model, ::Type{T}) where {T}
```

Comment:
Why this change ? Is it better like this ?

Reply:
`T::Type`, using the run-time value, which is a type.
`::Type{T}`: dispatching on `T`, which is a compile-time info

Reply:
Do you have a reference for that ? I thought I read that it recognizes this special case and it's equivalent

Reply:
so it's a special parsing rule for the compiler? I wasn't aware of one

Reply:
Not sure but I think the compiler might treat `Type` variables differently than other variables. Having to write `::Type{T}` everytime an argument is a type instead of just `T` or `T::Type` seems like working around something a compiler might do itself